src/app/components/ThemeSwitcher/index.tsx

```diff
@@ -19,9 +19,9 @@ import { getTargetTheme } from 'styles/theme/utils'
 interface Props {}
 
 const getThemesIcons = (t: TFunction) => ({
-  light: <Sun aria-label={t('theme.lightMode', 'Light mode')} size="24px" />,
-  dark: <Moon aria-label={t('theme.darkMode', 'Dark mode')} size="24px" />,
-  system: <System aria-label={t('theme.system', 'System')} size="24px" />,
+  light: <Sun aria-label={t('theme.lightMode', 'Light mode')} />,
```

Comment:
This works for `ThemeSwitcher`, but not for `ThemeSelect` which still needs `24px`. We have theme selection in a sidebar and in the profile settings modal (Grommet Select). Maybe we can add an optional size arg in `getThemesIcons` ?

Reply:
ooh that's why I changed it last time